.github/workflows/macos.yml

```diff
@@ -93,19 +93,16 @@ jobs:
         with:
            submodules: recursive
 
-      - name: install boost
+      - name: install dependencies
         run: |
           brew install boost-build boost boost-python3 python@3.9
-          export PYTHON_INCLUDE=`echo /usr/local/Cellar/python@3.9/*/Frameworks/Python.framework/Versions/3.9/include/python3.9` ;
-          echo "using darwin ;" >>~/user-config.jam
-          echo "using python : 3.9 : /usr/local/opt/python@3.9/bin/python3 : $(PYTHON_INCLUDE) : /usr/local/opt/python@3.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/config-3.9-darwin : <target-os>darwin ;" >> ~/user-config.jam
 
-      - name: build python module
+      - name: build and install python module
         run: |
           cd bindings/python
-          b2 -j2 stage_module stage_dependencies cxxstd=11
```

Comment:
the build is failing be cause `cxxstd=11` isn't passed to `b2` right now it seems

Reply:
oh, is `cxxstd=11` required on macos right now?

`setup.py` could detect and pass this, but should the jamfile enforce it instead? :thinking:

Reply:
in later version (`RC_2_0` and `master`) the Jamfile defaults to cxxstd=14 and cxxstd=17 respectively. I just haven't made that change to RC_1_2 because it supports being built with an old version of boost where the `cxxstd` option didn't exist yet